.github/actions/enable-cp-ingress-managed-vip/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: Enable CP Ingress VIP managed
+description: Enable the Control Plane Ingress managed Virtual IP
+
+runs:
+  using: "composite"
+  steps:
+    - name: Set Control Plane Ingress Managed VIP
+      uses: ./.github/actions/run-command-ssh
+      with:
+        COMMAND: >
+          sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf
+          patch clusterconfig main --type=merge --patch '{
+            \"spec\": {\"controlPlane\": {\"ingress\": {\"managedVirtualIP\": {\"address\": \"192.168.1.254\"}}}}
+          }'
+    - name: Wait for the Control Plane Ingress Managed VIP to be Ready
+      uses: ./.github/actions/run-command-ssh
+      with:
+        COMMAND: >
+          sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf
+          wait --for=condition=Ready ClusterConfig main --timeout=5m
+    - name: "Refresh the pillar"
+      uses: ./.github/actions/run-command-ssh
+      with:
+        COMMAND: >
+          sudo salt-call saltutil.refresh_pillar wait=True
+    - name: Reconfigure Bootstrap APIServer
+      uses: ./.github/actions/run-command-ssh
+      with:
+        COMMAND: >
+          sudo salt-call state.sls metalk8s.kubernetes.apiserver
+    - name: Reconfigure Control Plane components
+      uses: ./.github/actions/run-command-ssh
+      with:
+        COMMAND: |
+          SALT_MASTER=\$(sudo crictl ps --label="io.kubernetes.container.name=salt-master" -q)
+          sudo crictl exec \$SALT_MASTER salt-run state.sls metalk8s.orchestrate.update-control-plane-ingress-ip saltenv=metalk8s-$METALK8S_VERSION
```

Comment:
Depending on this `METALK8S_VERSION` env variable to be available is not ideal IMO, an input would make things clearer

Reply:
Yes but would always need input (that might be no longer needed at some point once (/if) we will have the full Ingress management in the operator) but right ok let's move it as input

Reply:
Done in https://github.com/scality/metalk8s/pull/4000/commits/4132813829c2f2aa465abfce7eeb247569111f11